diff -burNp python-bugzilla-0.3/bugzilla python-bugzilla-0.3-dud/bugzilla
--- python-bugzilla-0.3/bugzilla	2008-01-12 21:10:55.000000000 +0100
+++ python-bugzilla-0.3-dud/bugzilla	2009-03-14 23:16:02.402248815 +0100
@@ -14,8 +14,8 @@ import bugzilla, optparse
 import os, sys, glob, re
 import logging
 
-version = '0.2'
-default_bz = 'https://bugzilla.redhat.com/xmlrpc.cgi'
+version = '0.3'
+default_bz = 'https://bugs.pld-linux.org/xmlrpc.cgi'
 
 # Initial simple logging stuff
 logging.basicConfig()
